packages/node-runtime-worker-thread/src/rpc.ts

```diff
@@ -7,12 +6,20 @@ import {
   ServerMessageData,
   ClientMessageData
 } from 'postmsg-rpc';
+import {
+  serialize,
```

Comment:
are `serialize` and `serializeError` 2 separate functions cause we can't safely tell an error from another object?

Reply:
This, and also a bunch of properties on error are coming from prototype and non-enumerable, so they are just lost if we pass errors directly to `v8.serialize` without extracting them first